### PR TITLE
feat/ update readme with new rdf config

### PR DESCRIPTION
**Pull request by pj-cegeka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,25 +127,24 @@ The server allows configurable fragment refresh times with the max-age and max-a
...
-  collections:
-      - collection-name: { short name of the collection, cannot contain characters that are used for url interpretation, e.g. '$', '=' or '&' }
-        host-name: { hostname of LDES Server }
-        member-type: { Defines which syntax type is used to define the member id e.g. "https://data.vlaanderen.be/ns/mobiliteit#Mobiliteitshinder" }
-        timestamp-path: { SHACL property path to the timestamp when the version object entered the event stream. }
-        version-of: { SHACL property path to the non-versioned identifier of the entity. }
-        validation:
-          shape: { URI to defined shape }
-          enabled: { Enables/Disables shacl validation on ingested members }
+  host-name: { hostname of LDES Server }
```
Comment (Yalz) [inline README.md:133]: ldes-server prefix needed no?

**Review by Yalz — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -157,100 +156,174 @@ An example of a configuration containing multiple collections can be found [here
...
-  config:
-    maxZoomLevel: { Required zoom level }
-    fragmenterProperty: { Defines which property will be used for bucketizing }
+  ```ttl
+  @prefix ldes: <https://w3id.org/ldes#> .
+  @prefix tree: <https://w3id.org/tree#>.
+  @prefix example: <http://example.org/> .
+
+  <view1> a tree:Node ;
+    <https://w3id.org/tree#viewDescription> [
```
Comment (Yalz) [inline README.md:272]: can be replaced with tree:viewDescription

File: README.md [hunk 3]
```diff
@@ -157,100 +156,174 @@ An example of a configuration containing multiple collections can be found [here
...
-    memberLimit: { member limit > 0 }
+  ```ttl
+  @prefix prov: <http://www.w3.org/ns/prov#> .
+  @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+  @prefix ldes: <https://w3id.org/ldes#> .
+  @prefix tree: <https://w3id.org/tree#>.
+  @prefix example: <http://example.org/> .
+
+  <view1> a tree:Node ;
+    <https://w3id.org/tree#viewDescription> [
```
Comment (Yalz) [inline README.md:298]: can be replaced with tree:viewDescription

File: README.md [hunk 4]
```diff
@@ -127,25 +127,24 @@ The server allows configurable fragment refresh times with the max-age and max-a
...
-          enabled: { Enables/Disables shacl validation on ingested members }
+  host-name: { hostname of LDES Server }
   rest:
     max-age: { time in seconds that a mutable fragment can be considered up-to-date, default when omitted: 60 }
     max-age-immutable: { time in seconds that an immutable fragment should not be refreshed, default when omitted: 604800 }
   ```
 
-An example of a configuration containing multiple collections can be found [here](ldes-server-application/examples/multi-ldes-config-example.yml)
+The server can host one or more collections.
+These collection are to be added one by one on the following PUT endpoint.
```
Comment (Tomvbe) [inline README.md:140]: This was changed to a POST endpoint

**Pull request merged**